pyaedt/emit_core/__init__.py

```diff
@@ -7,8 +7,19 @@
 delcross_python_path = os.environ.get("ANSYS_DELCROSS_PYTHON_PATH")
 if delcross_python_path:
     sys.path.append(delcross_python_path)
-desktop_path = os.environ.get("ANSYSEM_ROOT232")
-if desktop_path and sys.version_info < (3, 8):
+desktop_path = os.environ.get("ANSYSEM_ROOT241")
+if not desktop_path:
+    desktop_path = os.environ.get("ANSYSEM_ROOT232")
+if desktop_path:
     path = os.path.join(desktop_path, "Delcross")
     sys.path.append(path)
-    EMIT_MODULE = import_module("EmitApiPython")
```

Comment:
@apollo-crum I don't think this change to the `import_module` call should be needed. We should still be able to `import_module("EmitApiPython")`. Instead of EmitApiPython referring to one EmitApiPython.pyd file, EmitApiPython is now a package (folder) in Delcross and the __init__.py in that folder takes care of importing the right version of pyd file (EmitApiPythonXY.pyd).